### ISSUE TITLE
How is plpythonu getting added to Postgres and ... 

### ISSUE THREAD

**Issue by NovaVic** [bcgov/ligo-lib#7]
https://github.com/bcgov-c/data-linking/blob/develop-2.0/lib/cdi-linking/db/algorithms

Review comment:
How is plpythonu getting added to Postgres? 
plpythonu is not safe either

About the SQL code in general
I participated in some of the conversations so I know what are these all about.
but considering other people who were not there and who may be looking at the code
 in future it would be a great idea to explain why these functions are here  or at the database level

Review comment: I would indent the code here (as for example) for the sake of readability: https://github.com/bcgov-c/data-linking/blob/develop-2.0/lib/cdi-linking/db/algorithms#L67 and other near by lines
 
_Reminder to self:_ during second iteration of review need to focus
on performance.
Also are the necessary indexes there even if needed temporarily? Double check would that be helpful?
Double check - why are we using both unix sort, pandas and postgres together
- basically why  the particular way of solving the problem has been picked.
